release-notes/major11/11.2.0/valtimo-backend-libraries.md

```diff
@@ -60,9 +68,10 @@ The following bugs were fixed:
 
   Solved a deadlock that could happen under certain conditions when modifying the document inside a BPMN process.
 
-* **Bug2**
+* **Fix process link 'Create zaakstatus**
```

Comment:
I remember discussing this in the past. This is the bugfixes section, so whatever we list here will be fixed. As such, the idea is to have a title that lets the reader know what the bug is, and if they want to know more they will read the description, where a more proper explanation of the issue is given. So no need to say a bug was fixed. If it wasn't fixed, it wouldn't be here in the list.